packages/mermaid/src/defaultConfig.ts

```diff
@@ -232,7 +234,7 @@ const config: Partial<MermaidConfig> = {
   },
   pie: {
     ...defaultConfigJson.pie,
-    useWidth: undefined,
+    useWidth: 984,
```

Comment:
It's not strange, it's the default one expected in cypress.

The width defined in pie is possibly `undefined`, so we use this value if it doesn't exists.


See https://github.com/mermaid-js/mermaid/pull/4501#discussion_r1296277525

Reply:
@Yokozuna59 please don't self-resolve comments like this one. Resolving [this comment](https://github.com/mermaid-js/mermaid/pull/4501#discussion_r1296513100) is fine, as you did an action that the reviewer was asking, and it was straightforward. 

In this case, the comment was a question, which you answered correctly. But you don't know if @nirname has seen the answer, or has any follow-up questions. The only way for @nirname to see your answer would be to click on each resolved comment to see which one it was. As you know, some PRs have many comments, and it would not be practical to go through each. :)

Leaving the comments unresolved makes the life of the reviewer easy. In case comments are left unresolved for a long time, other maintainers will handle it (By resolving it, or merging without resolving).

Reply:
@sidharthv96 Okay.

GitHub sends emails to subscribers when new actions occurs, so they'll be notified by an email.
And GitHub would also mark the PR/issue with blue circle showing that it has new actions.

:)

Reply:
> GitHub sends emails to subscribers when new actions occurs, so they'll be notified by an email.

Yeah, and in active PRs, there are many comments and emails going out. And Gmail sometimes collapses the thread. So I, personally, rarely check email for PR comments. 

>  And GitHub would also mark the PR/issue with blue circle showing that it has new actions.

But we won't know which comments were resolved recently when opening the PR. Everything would just be resolved and collapsed.